magefile.go

```diff
@@ -21,238 +21,21 @@
 package main
 
 import (
-	"errors"
-	"fmt"
-	"io"
-	"io/ioutil"
-	"log"
-	"net/http"
-	"os"
-	"path/filepath"
-	"strings"
-	"text/template"
-
+	// mage:import
```

Comment:
This tag is required by the doc if mage targets are imported https://magefile.org/importing/